cirq/ops/common_channels.py

```diff
@@ -184,6 +195,700 @@ def depolarize(p: float) -> DepolarizingChannel:
             the Pauli gates is applied independently with probability p / 3.
 
     Raises:
-        ValueError if p is not a valid probability.
+        ValueError: if p is not a valid probability.
     """
     return DepolarizingChannel(p)
+
+
+class GeneralizedAmplitudeDampingChannel(raw_types.Gate):
+    """Dampen qubit amplitudes through non ideal dissipation.
+
+    This channel models the effect of energy dissipation to the
+    surrounding environment at some particular rate, with some
```

Comment:
I think the main point about the Generalized amplitude damping channel is that it models both dissipation into the environment, and also the environment causing the system to get excited.  The reference to "temperature" is sort of reference to this later, but it's a bit confusing.  Would suggest:

This channel models the effect of energy dissipation into an environment as well 
as the environment depositing energy into the system.  The probabilities with which
the energy exchange occur are give by one constant, gamma, and the
probability of the environment being not excited is given by another constant,
p.

The stationary state of this channel is the diagonal density matrix with probability
p of being |0> and probability 1-p of being |1>.

Reply:
fixed, update to the docs to almost exactly what you gave here